src/components/MovieDetailsHeader/moviedetailsheader.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import './moviedetailsheader.css';
+import search from '../../img/search.svg';
+
+const MovieDetailsHeader = () => {
```

Comment:
Since it will be a part of the movie details page, I'd name it `MovieDetails`

Reply:
fixed